src/components/notifications/HdNotifications.vue

```diff
@@ -7,15 +7,15 @@
       <HdNotificationsBar
         v-for="(notification, i) in notifications"
         ref="notifications"
-        :key="notification.id"
-        :message="notification.message"
+        :key="notification.id || i"
         :type="notification.type"
-        :custom-icon="notification.customIcon || ''"
+        :message="notification.message"
+        :custom-icon="notification.customIcon"
         :visible="i === notifications.length - 1"
-        :offsetTop="offsetTop"
-        :offsetRight="offsetRight"
-        :offsetLeft="offsetLeft"
-      />
+        v-bind="$attrs"
```

Comment:
Yes, for now, it's just for `compact` but it can be used for other cases in the future where we need a common behavior in all the notifications. I just don't like declaring props in the parent and the child, especially setting the default value in two places.
I'm open regarding this point, I'll declare the prop explicitly.

Reply:
Hummmm.... I might have misunderstood something here. Does it make sense to have `compact` and `no compact` notifications simultaneously?
Nevertheless you could do to overload props as well (merging them before forwarding), might also be a solution.